homeassistant/components/flipr/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "domain": "flipr",
+  "name": "Flipr",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/flipr",
+  "requirements": [
+    "flipr-api==1.2.0",
+    "pycryptodome==3.10.1"],
+  "ssdp": [],
+  "zeroconf": [],
+  "homekit": {},
+  "dependencies": [],
```

Comment:
These are empty defaults that can be removed.

Reply:
OK, correction done via a new commit and push.